corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -460,7 +460,7 @@ def test_datasource_change_triggers_change_signal(self, datetime_mock, attempt_f
         transaction_log = transaction_logs[0]
         self.assertEqual(transaction_log.row_id, sample_doc["_id"])
         self.assertEqual(transaction_log.action, DataSourceRowTransactionLog.UPSERT)
-        self.assertEqual(attempt_forward_now_mock.call_count, 2)
+        create_repeat_records_mock.assert_called()
```

Comment:
is this to help with the unexpected error in tests through github actions?
Is this still testing the same thing?